packages/domain-lib/src/privileges.ts

```diff
@@ -33,9 +33,9 @@ export enum Privileges {
 	// CREATE_SETTLEMENT_BATCH = "SETTLEMENTS_CREATE_BATCH",
 	// CREATE_SETTLEMENT_TRANSFER = "SETTLEMENTS_CREATE_TRANSFER",
 
-	CREATE_DYNAMIC_SETTLEMENT_MATRIX = "SETTLEMENTS_CREATE_DYNAMIC_MATRIX",
-	CREATE_STATIC_SETTLEMENT_MATRIX = "SETTLEMENTS_CREATE_STATIC_MATRIX",
```

Comment:
is there no longer a need for static settlement matrices?

Reply:
Just to clarify, I saw the static matrix privilege removed and now just a general privilege to create a matrix. Does this mean there is now one privilege for both dynamic and static matrices?

Reply:
Yes, we will use the same Settlement create privilege for both static and dynamic matrix for creation.